guvnor-structure/guvnor-structure-client/src/test/java/org/guvnor/structure/client/editors/repository/clone/CloneRepositoryPresenterTest.java

```diff
@@ -306,192 +341,249 @@ public void testGitUrlValidation() {
+                                          repositoryEnvironmentConfigurationsArgumentCaptor.capture());
 
         final RepositoryEnvironmentConfigurations configurations = repositoryEnvironmentConfigurationsArgumentCaptor.getValue();
 
-        assertEquals( USERNAME, configurations.getUserName() );
-        assertEquals( PASSWORD, configurations.getPassword() );
-        assertEquals( REPO_URL, configurations.getOrigin() );
-        assertTrue( configurations.isManaged() );
+        assertEquals(USERNAME,
+                     configurations.getUserName());
+        assertEquals(PASSWORD,
+                     configurations.getPassword());
+        assertEquals(REPO_URL,
+                     configurations.getOrigin());
+        assertTrue(configurations.isManaged());
     }
 
     @Test
     public void testCloneNotManagedRepository() {
-        when( view.isGitUrlEmpty() ).thenReturn( false );
-        when( repositoryPreferences.isOUMandatory() ).thenReturn( false );
-        when( view.getGitUrl() ).thenReturn( REPO_URL );
-        when( view.getName() ).thenReturn( REPO_NAME );
-        when( view.isManagedRepository() ).thenReturn( false );
+        when(view.isGitUrlEmpty()).thenReturn(false);
+        when(repositoryPreferences.isOUMandatory()).thenReturn(false);
+        when(view.getGitUrl()).thenReturn(REPO_URL);
+        when(view.getName()).thenReturn(REPO_NAME);
+        when(view.isManagedRepository()).thenReturn(false);
 
         presenter.handleCloneClick();
 
-        verifyRepoCloned( true );
+        verifyRepoCloned(true);
 
-        verify( repoService, times( 1 ) ).createRepository( any( OrganizationalUnit.class ),
-                                                            anyString(),
-                                                            anyString(),
-                                                            repositoryEnvironmentConfigurationsArgumentCaptor.capture() );
+        verify(repoService,
+               times(1)).createRepository(any(OrganizationalUnit.class),
+                                          anyString(),
+                                          anyString(),
+                                          repositoryEnvironmentConfigurationsArgumentCaptor.capture());
         final RepositoryEnvironmentConfigurations configurations = repositoryEnvironmentConfigurationsArgumentCaptor.getValue();
 
-        assertEquals( USERNAME, configurations.getUserName() );
-        assertEquals( PASSWORD, configurations.getPassword() );
-        assertEquals( REPO_URL, configurations.getOrigin() );
-        assertFalse( configurations.isManaged() );
+        assertEquals(USERNAME,
+                     configurations.getUserName());
+        assertEquals(PASSWORD,
+                     configurations.getPassword());
+        assertEquals(REPO_URL,
+                     configurations.getOrigin());
+        assertFalse(configurations.isManaged());
+    }
+
+    @Test
```

Comment:
There are two tests for the cllone repository dialog. Otherwise it is formatting.